iron-core/src/main/java/io/axway/iron/core/spi/file/AtomicBigInteger.java

```diff
@@ -0,0 +1,27 @@
+package io.axway.iron.core.spi.file;
+
+import java.math.BigInteger;
+import java.util.*;
+import java.util.concurrent.atomic.*;
+
+public final class AtomicBigInteger {
+
+    private final AtomicReference<BigInteger> bigInteger;
+
+    public AtomicBigInteger(final BigInteger bigInteger) {
+        this.bigInteger = new AtomicReference<>(Objects.requireNonNull(bigInteger));
```

Comment:
code style : member bigInteger must be name m_bigInteger so "this." is not required

Reply:
Done (already fixed in the other PR).